### PR TITLE
hide logs and packages in get-build output if those are unavailable

### LINKED ISSUE #433: Show output of 'build-logs' in 'get-build'
Since we don't store logs in annotations anymore we should either show logs (same as in 'build-logs') or remove this section in 'get-build' output

### PR DESCRIPTION
**Pull request by vrutkovs**
Fixes #433

### CODE REVIEW

File: osbs/cli/main.py [hunk 1]
```diff
@@ -206,15 +223,7 @@ def cmd_get_build(args, osbs):
...
-
-            BUILD LOGS
-
-            {logs}
-
-            PACKAGES
-
-            {packages}
-
+            {logs}{packages}
```
Comment (lcarva) [inline osbs/cli/main.py]: Are we missing newlines here? `"""\` won't add a new line.

Comment (vrutkovs) [inline osbs/cli/main.py]: I'm not sure why but it seems each empty line is rendered with ```\n```, see [sample output](https://gist.github.com/vrutkovs/dffd745242230137e0e3346b12a08259).

File: osbs/build/build_response.py [hunk 2]
```diff
@@ -97,7 +97,7 @@ def get_logs(self, decode_logs=True):
         """
         logs = graceful_chain_get(self.get_annotations_or_labels(), "logs")
         if not logs:
-            logger.error("no logs")
+            logger.warn("no logs found in annotations")
```
Comment (mmilata) [inline osbs/build/build_response.py]: If we expect this to normally happen with future builds then let's use some other log level - debug?

**Comment by mmilata** [comment]
Otherwise LGTM

File: osbs/build/build_response.py (hunk 2 above)
Comment (vrutkovs) [inline osbs/build/build_response.py]: Agreed, will change that

**Comment by mmilata** [comment]
LGTM, merged, thank you.

**Pull request merged**